test/docker/docker.go

```diff
@@ -71,15 +71,15 @@ func (d *DockerCompose) Start(ctx context.Context, container string) error {
 }
 
 func (d *DockerCompose) StopAt(ctx context.Context, nodeIndex int, timeout *time.Duration) error {
-	if nodeIndex > len(d.containers) {
-		return fmt.Errorf("container at index %d does not exit", nodeIndex)
+	if nodeIndex >= len(d.containers) {
+		return nil
```

Comment:
**suggestion (if-minor):** do we wanna keep this to detect if there was an issue ?

Reply:
This will require a number of changes in several tests, as the `stopNodes` function attemps to stop all containers, whether or not any given container should be up at that moment in time. 

Agreed that this should be done, think we can just do it at a later time